cms/djangoapps/contentstore/exams.py

```diff
@@ -0,0 +1,123 @@
+"""
+Code related to working with the exam service
+"""
+
+import json
+import logging
+
+import requests
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from edx_rest_api_client.auth import SuppliedJwtAuth
+
+from openedx.core.djangoapps.course_apps.toggles import exams_ida_enabled
+from openedx.core.djangoapps.oauth_dispatch.jwt import create_jwt_for_user
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+from .views.helpers import is_item_in_course_tree
+
+log = logging.getLogger(__name__)
+User = get_user_model()
+
+
+def register_exams(course_key):
```

Comment:
there is some intentional code duplication between this function and the edx-proctoring `register_special_exams`. I think it would be messier to try to reuse some of these statements when these two integrations may evolve differently.